packages/web/components/Dashboard/Settings/SubscriptionForm.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react'
+import { useForm } from 'react-hook-form'
+import { toast } from 'react-toastify'
+import { loadStripe } from '@stripe/stripe-js'
+import { Elements, CardElement, useStripe, useElements } from '@stripe/react-stripe-js'
+import Button from '@/components/Button'
+import FormError from '@/components/FormError'
+import { useTranslation } from '@/config/i18n'
+import SettingsForm from '@/components/Dashboard/Settings/SettingsForm'
+import SettingsFieldset from '@/components/Dashboard/Settings/SettingsFieldset'
+import nProgress from 'nprogress'
+
+type FormValues = {
+  type: UserSubscriptionType
```

Comment:
Where `UserSubscriptionType` is coming?

Reply:
This will be `MembershipSubscriptionType` later which originally comes from the new enum that you can find in `schema.prisma` (was originally called `UserSubscriptionType` in the first first draft but I wanted to differentiate it more from other types of "Subscription"). We'll have one client-side from our Apollo codegen